llama_stack/providers/inline/agents/meta_reference/persistence.py

```diff
@@ -183,21 +183,39 @@ async def get_num_infer_iters_in_turn(self, session_id: str, turn_id: str) -> in
         )
         return int(value) if value else None
 
-    async def list_sessions(self) -> list[Session]:
-        values = await self.kvstore.values_in_range(
```

Comment:
by removing this, you are making this extremely inefficient right?

Reply:
Reverted to `values_in_range` and skipping `turn` keys. 👍